lib/contacts.py

```diff
@@ -63,6 +64,14 @@ def import_file(self, path):
         self.update(d)
         self.save()
 
+    def export_file(self, fileName):
+        try:
+            with open(fileName, 'w+') as f:
+                json.dump(self, f, indent=4, sort_keys=True)
+        except (IOError, os.error) as reason:
+            traceback.print_exc(file=sys.stderr)
+            raise reason
```

Comment:
Not sure how much sense it makes to catch these exception and just re-raise them like this, here.
Could you maybe do it similar to `import_file`? Define `FileExportFailed` and raise that here.